### PR TITLE
offline running

### PR DESCRIPTION
**Pull request by shuishen112**
I add the offline running features and I hope it will be helpful when someone has the same problems.

### CODE REVIEW

File: pyterrier/__init__.py [hunk 1]
```diff
@@ -63,9 +64,12 @@ def init(version=None, mem=None, packages=[], jvm_opts=[], redirect_io=True, log
      * If the `version` init kwarg is not set, Terrier will query MavenCentral to determine the latest Terrier release.
      * If `version` is set to `"snapshot"`, the latest .jar file build derived from the `Terrier Github repository <https://github.com/terrier-org/terrier-core/>`_ will be downloaded from `Jitpack <https://jitpack.io/>`_.
      * Otherwise the local (`~/.mvn`) and MavenCentral repositories are searched for the jar file at the given version.
-     
     In this way, the default setting is to download the latest release of Terrier from MavenCentral. The user is also able to use a locally installed copy in their private Maven repository, or track the latest build of Terrier from Jitpack.
     
+    If your running enviroment is offline. you can download the "terrier-assemblies-{your version}-jar-with-dependencies.jar" and "terrier-python-helper-{your helper version}.jar"
```
Comment (cmacdonald) [inline pyterrier/__init__.py:69]: Its enough to run pyterrier online, and then to set the versions for the offline scenario, right?

Comment (shuishen112) [reply pyterrier/__init__.py:69]: Yes, you need to set the versions and helper_version so that you can run offline.

**Comment by cmacdonald** [comment]
Hi @shuishen112 can I have a name (and affiliation) for the changelog?
Many thanks in advance,

**Comment by shuishen112** [comment]
> Hi @shuishen112 can I have a name (and affiliation) for the changelog?
> Many thanks in advance,

Thanks, my name is Zhan Su and I am a new PhD student at Copenhagen University.

**Pull request merged**